TrelloAutotest/Tests/ManageBoardsTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NUnit.Framework;
+using TrelloAutotest.Pages;
+using TrelloAutotests.Tests;
+
+namespace TrelloAutotest.Tests
+{
+    class ManageBoardsTest : BaseTest
+    {
+        private TrelloMainPage _trelloMainPage;
+        private TrelloCreateBoardPage _configureBoard;
+        private TrelloBoardPage _boardPage;
+        private const string _boardName = "Nowa tablica ";
+
+        [Test]
+        public void CreateNewBoard()
+        {
+            _trelloMainPage = new TrelloMainPage();
+            _configureBoard = new TrelloCreateBoardPage();
+            _boardPage = new TrelloBoardPage();
+
+            _trelloMainPage.CreateNewBoard();
+            _configureBoard.CreateBoard(_boardName);
+          
+            Assert.IsTrue(_boardPage.IsUrlCorrect(_boardName));
```

Comment:
Probably unawarely, but you're following here AAA pattern, which is nice! :) We're using it mainly for unit tests, but they are good for automation too.